packages/block-library/src/embed/embed-preview.native.js

```diff
@@ -53,19 +59,64 @@ const EmbedPreview = ( {
 		}
 	}
 
+	const { scripts, provider_url: providerUrl } = preview;
+	const html = 'photo' === type ? getPhotoHtml( preview ) : preview.html;
+	const parsedHost = new URL( url ).host.split( '.' );
+	const parsedHostBaseUrl = parsedHost
+		.splice( parsedHost.length - 2, parsedHost.length - 1 )
+		.join( '.' );
+	const iframeTitle = sprintf(
+		// translators: %s: host providing embed content e.g: www.youtube.com
+		__( 'Embedded content from %s' ),
+		parsedHostBaseUrl
+	);
+
+	const embedWrapper =
+		/* We should render here: <WpEmbedPreview html={ html } /> */
+		'wp-embed' === type ? null : (
+			<>
+				<TouchableWithoutFeedback
+					onPress={ () => {
+						if ( onFocus ) {
+							onFocus();
+						}
+						if ( isCaptionSelected ) {
+							setIsCaptionSelected( false );
+						}
+					} }
+				>
+					<View pointerEvents="box-only">
+						<SandBox
+							html={ html }
+							scripts={ scripts }
```

Comment:
Eventually, we might remove this prop for the mobile version because I noticed that for some scripts the path is relative and since we're not in the web version of the editor, these scripts won't load, here is an example of the scripts passed for a Giphy embed:
```
0: "/wp-content/js/devicepx.js"
1: "/wp-content/js/postmessage.js"
2: "/wp-content/js/mobile-useragent-info.js"
3: "/wp-content/mu-plugins/subscriptions/follow.js"
```

Reply:
Could these maybe be needed for the mobile editor as well?

Reply:
I haven't investigated further what these scripts do but as far as I tested, the previews were properly rendered without them. Anyways, I'll check deeper just in case we're missing anything.

Reply:
Looks like the base URL for those scripts should be `https://wordpress.com`:
- https://wordpress.com/wp-content/js/devicepx.js
- https://wordpress.com/wp-content/js/postmessage.js
- https://wordpress.com/wp-content/js/mobile-useragent-info.js
- https://wordpress.com/wp-content/mu-plugins/subscriptions/follow.js

However, there's no reference to them in the `oembed` response so I'm not sure how to infer it, one option would be to use `https://wordpress.com` as a fallback when there's no provider URL, wdyt?

Reply:
I think maybe we shouldn't provide a fallback here, but skip the injection instead. Although in this case I don't see where `wordpress.com` comes from when the path is relative (to the website itself), how does the web version infer them? Is it because the editor is running on a wpcom hosted site in this case?

Reply:
> Although in this case I don't see where wordpress.com comes from when the path is relative (to the website itself), how does the web version infer them? Is it because the editor is running on a wpcom hosted site in this case?

Yeah, in the web version the relative paths are resolved with the browser's current location, which for WPcom sites would be `wordpress.com`. That's the main difference with the native version, since we're not in a webpage but using static HTML we have to provide a base URL for this.

On self-hosted sites, as far as I checked, the providers that require scripts are displayed as not supported 😅 . 

> I think maybe we shouldn't provide a fallback here, but skip the injection instead.

I think both would work but it's true that adding a fallback with a static string might be not the best option, being that said, I'll remove the script injection 👍 .

Reply:
In [this commit](66c03be03a69c0e425b4511ec947a66ca2057ab0) I removed the script injection in the sandbox component.